cmd/habitat/main.go

```diff
@@ -33,6 +35,8 @@ var (
 
 func main() {
 	pflag.String("procdir", "", "directory where process configs are stored")
+	pflag.String("communitydir", "", "directory where communities are stored")
+	pflag.String("hostname", "", "hostname that this node can be reached at")
```

Comment:
this is the hostname for raft right? i think we should be clear about that

Reply:
The hostname is a general networking thing. We might not need this flag because of the compass lib tho.